### PR TITLE
[Chat] Updated CC flags for File Sharing GA Release

### PR DESCRIPTION
**Pull request by jpeng-ms**
# What

For upcoming file sharing GA release for Teams interop chat, we are making 2 APIs public so contoso can build interop experience using components. For composite, the file sharing support is built-in with no configurations required.

2 APIs we are releasing as GA are:
- AttachmentMetadata
- ChatMessage.attachments

# Why

GA release

# How Tested

Sample app

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/theming/icons.tsx [hunk 1]
```diff
@@ -85,8 +85,10 @@ import { Backspace20Regular } from '@fluentui/react-icons';
 /* @conditional-compile-remove(call-readiness) */
 import { Sparkle20Filled, VideoProhibited20Filled, MicProhibited20Filled } from '@fluentui/react-icons';
 
-/* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
-import { ArrowDownload20Regular, Open20Regular } from '@fluentui/react-icons';
+/* @conditional-compile-remove(file-sharing-teams-interop) */
+import { Open20Regular } from '@fluentui/react-icons';
```
Comment (jimchou-dev) [inline packages/react-components/src/theming/icons.tsx:89]: should this have both CC flag for interop and acs?

File: packages/react-components/src/types/Attachment.ts [hunk 2]
```diff
@@ -4,13 +4,16 @@
...
 /**
  * @beta
  *
  * Attachment Options that defines behaviour for uploading and downloading attachments.
  */
 export interface AttachmentOptions {
+  /* @conditional-compile-remove(file-sharing-acs) */
   uploadOptions?: AttachmentUploadOptions;
+  /* @conditional-compile-remove(file-sharing-acs) */
   downloadOptions?: AttachmentDownloadOptions;
```
Comment (jimchou-dev) [inline packages/react-components/src/types/Attachment.ts:17]: can we just cc flag for the whole thing and not individually for upload/download options?

Comment (jpeng-ms) [reply packages/react-components/src/types/Attachment.ts]: thanks updated

**Review by emlynmac — APPROVED** [review]
It would be good to separate out functional changes and non-functional changes.

Given the time constraints, this is approved.

**Review by palatter — APPROVED** [review]

**Review by jimchou-dev — APPROVED** [review]

**Review by Leah-Xia-Microsoft — APPROVED** [review]

**Pull request merged**